### ISSUE TITLE
Running tx.outputs.address.type crashes jupyter kernel on 0.5 ami

### ISSUE THREAD

**Issue by Haaroon** [citp/BlockSci#324]
Please provide a clear and concise description of the problem.

#### Reproduction Steps

```
import blocksci
import matplotlib.pyplot as plt
import matplotlib.ticker
import collections
import pandas as pd
import numpy as np
%matplotlib notebook
chain = blocksci.Blockchain("/home/ubuntu/bitcoin")
def filter_pwshout(block):
    vals = block.net_address_type_value()
    if blocksci.address_type.witness_scripthash in vals.keys():
        return block
    
pwshtxs = [block for block in chain.map_blocks(blockFunc=filter_pwshout) if block != None]
b = pwshtxs[0]
tx = b.txes[10]
tx.outputs.address.type
```

#### System Information

Using AMI: yes
BlockSci version: 0.5
Blockchain: Bitcoin
Parser: Disk
Total memory: lots  of GB  

😢

**Comment by Haaroon** [comment]
if you do this, so basically wrap the addresses in a list, then check their individual types, it doesnt crash 

```
def extract_pwsh_txs(blocks):
    count = 0
    s = blocksci.address_type.witness_scripthash
    correct = []
    for b in blocks:
        count += 1
        if count % 100 == 0:
            print('%i/%i' % (count, len(blocks)))
        for tx in b.txes:
            addrs = list(tx.outputs.address)
            for addr in addrs:
                if addr.type == blocksci.address_type.witness_scripthash:
                    correct.append(tx)
                    break
    return correct

pwsh_txs = extract_pwsh_txs(pwshtxs)
```

**Comment by martinplattnr** [comment]
The code from your initial comment does not crash on my v0.6-based instance.
The output is `[address_type.pubkeyhash, address_type.pubkeyhash]`.

On my v0.5 instance the code is still running, will report back if it crashed.

**Comment by martinplattnr** [comment]
Update: Using v0.5, the kernel crashed, while v0.6 worked fine.
So I guess this issue can be labelled `fixed-in-v0.6`, @maltemoeser.

**Comment by Haaroon** [comment]
Thank you for confirming, i shall update to v0.6.

Haaroon
